python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ip_name_server_groups.schema.yml

```
@@ -42,3 +42,12 @@ keys:
         ip_domain_list:
           type: str
           description: Set domain names to complete unqualified host names.
+          deprecation:
+            warning: true
+            new_key: ip_name_server_groups[].ip_domain_lists
```

I think with this it will help trigger validation error when both keys are set
```suggestion
            new_key: ip_domain_lists
```

---

Done